### ISSUE TITLE
High contrast map needs matching UI elements

### ISSUE THREAD

**Issue by tituomin** [City-of-Helsinki/servicemap#331]

**Comment by kallelongjuhani** [comment]
See here:
https://drive.google.com/open?id=0Bz_jZaEUKzZ9WXFkUlZSVG5ScE0

This may still require tweaks to the background map colours, but let's implement these changes first and see, if those are needed.

**Comment by vaaralav** [comment]
@kallelongjuhani is there high contrast versions of Servicemap logos (=with the outline) somewhere?

**Comment by kallelongjuhani** [comment]
@vaaralav no, I mistakenly thought those could be added in CSS :) Do you need png images or are svgs enough?

**Comment by vaaralav** [comment]
Png images would be better :)

**Comment by kallelongjuhani** [comment]
Hi! Png's here: https://drive.google.com/open?id=0Bz_jZaEUKzZ9NFFTZkRBbmFEaEk 

You can download the zip "high-contrast-logos-(outlined).zip" for convenience.
